### DISCUSSION TITLE
Optimizing Next js

### DISCUSSION THREAD

**Discussion by sanjaiyan-dev** [vercel/next.js#31204] [Help] [answered]
### Describe the feature you'd like to request

Hi, I tested my website which is developed using next js 12.0.3(next export) in [Google Web Dev](https://web.dev/measure) and I got following suggestion.

![Screenshot_2021-11-08-17-42-37-068_com android chrome](https://user-images.githubusercontent.com/88643231/140912428-19c9b6b5-bb97-4359-bab4-142deb3da304.png)

### Describe the solution you'd like

Using `renderToNodeStream` and `renderToStaticNodeStream` for performance optimization.

### Describe alternatives you've considered

Sorry I don't know much about, React and next js so sorry if I made any mistakes.

**Comment by housseindjirdeh** [comment, marked as answer]
Please don't apologize! You didn't make any mistake, this is a good question :)

The React suggestions in Lighthouse were written a few years ago and were focused on providing audit suggestions for React developers building a "custom" stack (setting up their own server, etc...). Next.js already includes server-side rendering by default and optimizes client-rendering and server-rendering as much as possible, so this message isn't relevant for your use-case and you don't need to worry about it.

@kyliau is working on introducing Next.js-specific messaging to Lighthouse, so once that rolls out that should hopefully clear up some confusion.

**Reply by sanjaiyan-dev** [reply]
Thanks a lot for clearing my doubt and for your kind words :)

**Reply by iankduffy** [reply]
Will the updated lighthouse affect the performance scores, or will it just be the messages for improvements?

**Reply by kyliau** [reply]
Just the messages. Lighthouse Stack Packs does not perform additional audits.
